deploy/Chart/templates/controller/rbac.yaml

```diff
@@ -11,6 +11,7 @@ rules:
   resources:
   - namespaces
   - secrets
+  - events
```

Comment:
Needed for diagnostics. This is the best way to enable troubleshooting for users.